x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_table.tsx

```diff
@@ -242,10 +243,8 @@ export const RulesListTable = (props: RulesListTableProps) => {
         hideSnoozeOption
         disableRule={async () => await onDisableRule(rule)}
         enableRule={async () => await onEnableRule(rule)}
-        snoozeRule={async (snoozeEndTime: string | -1, interval: string | null) => {
-          await onSnoozeRule(rule, snoozeEndTime);
-        }}
-        unsnoozeRule={async () => await onUnsnoozeRule(rule)}
+        snoozeRule={async () => {}}
+        unsnoozeRule={async () => {}}
```

Comment:
I wasn't able to figure out how to get Typescript to only require these props if `hideSnoozeOption` is false, just keeping them here turned out to be easier. I assume we're planning on deprecating the snooze option in the dropdown in a later PR, though, so that might be a good time to do that.